### ISSUE TITLE
Friendly error message toggle 

### ISSUE THREAD

**Issue by rismoney** [voxpupuli/puppetboard#688]
I believe this is not pointing to a proper url

```
the requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
```

Testing has been docker using docker 3.5 container.
looks like the /raw isn't found

**Comment by gdubicki** [comment]
Thanks for the report @rismoney!

Can you please provide an exact step-by-step how to recreate it? Because I did some tests and did not have the issue.

**Comment by gdubicki** [comment]
It turned out to be trivial to reproduce, I didn't need any extra info. Fixed in v3.5.1 released a moment ago.
